support/android/apk/servoapp/src/main/AndroidManifest.xml

```diff
@@ -26,6 +27,7 @@
               <category android:name="android.intent.category.DEFAULT" />
               <category android:name="android.intent.category.BROWSABLE" />
               <data android:scheme="http" />
+              <data android:host="*"/>
```

Comment:
What is the motivation for this change? Was it not working previously?

Reply:
It was a part of a series of multiple commits addressing android lint errors, which in turn caused the CI to fail.

You may experiment on a local machine by dropping the commits and running the android lint task from gradle.

Reply:
Checking in, Have you done the experiment yet?

Reply:
Sorry about the delay. I did try removing `host` locally and it didn't fail the build for me. Do you have links to the logs from CI where it failed?

Reply:
I did a [test run on the CI ](https://github.com/mukilan/servo/actions/runs/11625426495) without the host directives and it succeeded.  I'll update this PR to remove those. We can always add it back if we find it is needed.